### ISSUE TITLE
Question: Does the plugin support Bluetooth classic?

### ISSUE THREAD

**Issue by DeepakArora76** [EddyVerbruggen/nativescript-bluetooth#73]
Does the plugin support Bluetooth classic? Or it for devices which support Bluetooth Low Energy (BLE)?

Is it possible to use the plugin between two paired devices (iOS-iOS, Android-Android, Android-iOS)?

To me from the source code of Android, it appears that the plugin support only BLE.

Appreciate your inputs.

**Comment by EddyVerbruggen** [comment]
Yeah, and that’s also in the readme.

**Comment by DeepakArora76** [comment]
I do not find any references to Bluetooth Classic or BLE in the **Readme** document.

From the link **https://www.npmjs.com/package/nativescript-bluetooth**  what is see that:

Use when you want to
    - **scan for Bluetooth LE / Smart peripherals**,
    - **connect to those peripherals**,
    - read values of a characteristic,
    - write new values to them,
    - get notified when the value of characteristic changes.

**So, in a nutshell, Bluetooth classic is not supported by the plugin, right?**

**Comment by EddyVerbruggen** [comment]
BLE only.

<img width="591" alt="schermafbeelding 2018-02-16 om 13 40 28" src="https://user-images.githubusercontent.com/1426370/36307986-fc094e18-131e-11e8-84b0-81352e762d7a.png">
